Assets/MRTK/Core/Providers/InputSimulation/InputSimulationService.cs

```diff
@@ -425,6 +425,11 @@ private void EnableCameraControl()
             if (cameraControl == null)
             {
                 cameraControl = new ManualCameraControl(InputSimulationProfile);
+            
+                if (CameraCache.Main)
```

Comment:
slight nit, but we've tended to be explicit with null checks in this repo:

```suggestion
                if (CameraCache.Main != null)
```